docs/rest_api/emails.rst

```diff
@@ -430,6 +445,12 @@ Create a new Email.
    * - ``bccAddress``
      - string
      - The ``BCC`` Email address if it's different than the one in the Mautic configuration
+   * - ``useOwnerAsMailer``
+     - boolean
+     - Whether or not contact owner should be used as mailer. Defaults to TRUE
```

Comment:
Fix the Vale warning about Contact being capitalised, and a suggestion for the tense.

```suggestion
     - Should Mautic use the Contact owner as the sender of the Email. Defaults to TRUE
```